rediscluster/client.py

```diff
@@ -71,6 +72,20 @@ class StrictRedisCluster(StrictRedis):
         ], 'slot-id'),
     )
 
+    # Not complete, but covers the major ones
+    # https://redis.io/commands
+    READ_COMMANDS = Set(["BITPOS", "BITCOUNT", "EXISTS",
```

Comment:
Using a `Set` object here might be less good as it would not allow for a user of the lib to add new commands that is not yet put into here by the lib in a release. A plain list would be good enough.